tests/fe/fe_nedelec_face_edge_orientation_0.cc

```diff
@@ -0,0 +1,28 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2013 - 2023 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+#include "fe_nedelec_face_edge_orientation.h"
+
+int
+main()
+{
+  unsigned char combined_face_orientation = 0;
+
+  initlog();
+  run<3, 0>(combined_face_orientation);
```

Comment:
All the tests you are adding here do the same thing for different orientations. Would you mind making them a single test with a ``for`` loop for all cases?

Reply:
I do not mind to put all tests into one file. Such aggregated test, however, will run longer than 180 sec  which is the default time limit for a test, see dealii/doc/developers/testsuite.html. Do you think it makes sense to put all tests in one file knowing that the run time will be longer than the default time limit?  I am not sure what is the common practice. An inspection of the test files leaves an impression that everybody tries to avoid exceeding the default time limit.

Reply:
These are the run times of the current version of the tests:

Test project /home/siarhei/dealii-9.6.0-nedelec-build
      Start 2513: fe/fe_nedelec_face_edge_orientation_0.debug
      Start 2514: fe/fe_nedelec_face_edge_orientation_0.release
      Start 2515: fe/fe_nedelec_face_edge_orientation_1.debug
      Start 2516: fe/fe_nedelec_face_edge_orientation_1.release
      Start 2517: fe/fe_nedelec_face_edge_orientation_2.debug
      Start 2518: fe/fe_nedelec_face_edge_orientation_2.release
      Start 2519: fe/fe_nedelec_face_edge_orientation_3.debug
      Start 2520: fe/fe_nedelec_face_edge_orientation_3.release
 1/18 Test #2516: fe/fe_nedelec_face_edge_orientation_1.release ...   Passed   21.99 sec
      Start 2521: fe/fe_nedelec_face_edge_orientation_4.debug
 2/18 Test #2514: fe/fe_nedelec_face_edge_orientation_0.release ...   Passed   22.00 sec
      Start 2522: fe/fe_nedelec_face_edge_orientation_4.release
 3/18 Test #2520: fe/fe_nedelec_face_edge_orientation_3.release ...   Passed   22.03 sec
      Start 2523: fe/fe_nedelec_face_edge_orientation_5.debug
 4/18 Test #2518: fe/fe_nedelec_face_edge_orientation_2.release ...   Passed   22.39 sec
      Start 2524: fe/fe_nedelec_face_edge_orientation_5.release
 5/18 Test #2522: fe/fe_nedelec_face_edge_orientation_4.release ...   Passed   19.72 sec
      Start 2525: fe/fe_nedelec_face_edge_orientation_6.debug
 6/18 Test #2524: fe/fe_nedelec_face_edge_orientation_5.release ...   Passed   20.01 sec
      Start 2526: fe/fe_nedelec_face_edge_orientation_6.release
 7/18 Test #2526: fe/fe_nedelec_face_edge_orientation_6.release ...   Passed   19.34 sec
      Start 2527: fe/fe_nedelec_face_edge_orientation_7.debug
 8/18 Test #2513: fe/fe_nedelec_face_edge_orientation_0.debug .....   Passed   79.24 sec
      Start 2528: fe/fe_nedelec_face_edge_orientation_7.release
 9/18 Test #2519: fe/fe_nedelec_face_edge_orientation_3.debug .....   Passed   79.68 sec
      Start 2529: fe/fe_nedelec_face_edge_orientation_8.debug
10/18 Test #2515: fe/fe_nedelec_face_edge_orientation_1.debug .....   Passed   79.85 sec
      Start 2530: fe/fe_nedelec_face_edge_orientation_8.release
11/18 Test #2517: fe/fe_nedelec_face_edge_orientation_2.debug .....   Passed   79.86 sec
12/18 Test #2528: fe/fe_nedelec_face_edge_orientation_7.release ...   Passed   17.16 sec
13/18 Test #2523: fe/fe_nedelec_face_edge_orientation_5.debug .....   Passed   75.66 sec
14/18 Test #2521: fe/fe_nedelec_face_edge_orientation_4.debug .....   Passed   75.73 sec
15/18 Test #2530: fe/fe_nedelec_face_edge_orientation_8.release ...   Passed   18.29 sec
16/18 Test #2529: fe/fe_nedelec_face_edge_orientation_8.debug .....   Passed   19.94 sec
17/18 Test #2525: fe/fe_nedelec_face_edge_orientation_6.debug .....   Passed   67.08 sec
18/18 Test #2527: fe/fe_nedelec_face_edge_orientation_7.debug .....   Passed   57.28 sec